src/components/Form/FormHelpButton.tsx

```diff
@@ -36,8 +42,8 @@ export default function FormHelpButton({ identifier, label, help }: FormHelpButt
       <UncontrolledPopover placement="right" target={safeId(identifier)} trigger="legacy" hideArrow>
         <Card className="card--help">
           <CardBody>
-            {label && <h4>{onlyText(label)}</h4>}
-            <p>{help}</p>
+            <h4>{help.label}</h4>
```

Comment:
Now we no longer need to invoke `onlyText`. We just insert the contents of the 'label' field from the HelpProps object.